### PR TITLE
[breaking] changing driver api; adding populateCache()

### PR DESCRIPTION
**Pull request by nemtsov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,15 +16,15 @@ This Browser / Node.js library is a helper for loading mustache templates via Ho
...
-const compiler = hoganCompiler.create(fsReadFile, templatesPath/*, options */);
+const compiler = hoganCompiler.create(fsDriver, templatesPath/*, options */);
 
-compiler.addLambda('i18n', () => {
-  return (text) => {
-    return my_18n_library.tranlate(text);
-  };
+// When using `options.isCached`, you may want to populate the cache before making `.compile()` calls
+compiler.populateCache()
+.then(() => {
```
Comment (agaripian) [inline README.md]: single line please

Comment (nemtsov) [inline README.md]: but but but but

Comment (nemtsov) [inline README.md]:
```js
compiler.populateCache().then(() => console.log('Cache has been populated'));
```
?

Comment (agaripian) [inline README.md]: yes

**Comment by agaripian** [comment]
👍

**Pull request merged**